src/lib/types/modals.ts

```diff
@@ -0,0 +1,69 @@
+//- Features Imports
```

Comment:
Same advice as for `ModalProvider` - this is probably going to get really hectic as we add more features. Best to think of a more generic way to write this.

Reply:
Agreed. Using this modal provider was adding unnecessary complexity. Thanks for highlighting the potential issues.

Removed modals.ts.